tests/unit/Debugger/DaemonTest.php

```diff
@@ -114,6 +114,27 @@ public function testDefaultSourceContext()
         ]);
     }
 
+    public function testDefaultSourceContext()
+    {
+        $expectedSourceContext = [
+            'context' => [
+                'git' => [
+                    'revisionId' => '81b20d097da02ebb6c6fdfbf6900c67a90f2c54b',
+                    'url' => 'https://github.com/GoogleCloudPlatform/google-cloud-php.git'
+                ]
+            ]
+        ];
+        $this->debuggee->register(Argument::any())->shouldBeCalled();
+        $this->client->debuggee(null, Argument::withEntry('extSourceContexts', [$expectedSourceContext]))
+            ->willReturn($this->debuggee->reveal())->shouldBeCalled();
+
+        $root = implode(DIRECTORY_SEPARATOR, [__DIR__, 'example']);
```

Comment:
Nit: The directory name `example` seems weird to me. How about `data`?

Reply:
Sure, we can change it.